python/ray/serve/tests/storage_tests/test_kv_store.py

```diff
@@ -128,7 +128,6 @@ def put(self, key: str, val: bytes) -> bool:
         return super().put(key, val)
 
 
-@pytest.mark.skipif(sys.platform == "win32", reason="Using tmp dir.")
 def test_make_kv_store(serve_instance):
```

Comment:
I didn't see any failure because of using tmp dir. Is it due to having administrator access (because I have one on my cloud machine)?